kroxylicious-operator/pom.xml

```diff
@@ -327,4 +327,143 @@
             </testResource>
         </testResources>
     </build>
+
+    <profiles>
+        <profile>
+            <id>kubeTest</id>
+            <properties>
+                <container.tool>podman</container.tool> <!-- override for docker -->
+                <container.tag>test</container.tag>
+                <image.archive.file>/tmp/myimage.tar</image.archive.file>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>3.5.0</version>
+                        <executions>
+                            <execution>
+                                <!-- build the container image -->
+                                <id>podman-build</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <workingDirectory>..</workingDirectory>
+                                    <executable>${container.tool}</executable>
+                                    <arguments>
+                                        <argument>build</argument>
+                                        <argument>--file=./Dockerfile.operator</argument>
+                                        <argument>--build-arg=KROXYLICIOUS_VERSION=${project.version}</argument>
+                                        <argument>--tag=kroxylicious.io/operator:${container.tag}</argument>
+                                    </arguments>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <!-- export the container image to a temp file -->
+                                <id>podman-push</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <workingDirectory>..</workingDirectory>
+                                    <executable>${container.tool}</executable>
+                                    <arguments>
+                                        <argument>push</argument>
+                                        <argument>kroxylicious.io/operator:${container.tag}</argument>
+                                        <argument>docker-archive:${image.archive.file}</argument>
```

Comment:
This failed on my apple silicone mac with
```
[INFO]
[INFO] --- exec:3.5.0:exec (podman-push) @ kroxylicious-operator ---
Error: unsupported transport docker-archive in "docker-archive:/tmp/myimage.tar": only docker transport is supported
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 125 (Exit value: 125)
```

Using 
`podman version 5.4.2`

Reply:
Now replaced with a fabric8 thingy. Can you try again and let me know where it fails now?

Reply:
Easiest to test with `mvn -P dist,containers,kubeTest install -rf :kroxylicious-operator -DskipITs` btw

Reply:
Works for me with Apple with Intel chipset

Reply:
For the record: starts have aligned 
```
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:53 min
[INFO] Finished at: 2025-05-12T11:53:37+12:00
[INFO] ------------------------------------------------------------------------
```